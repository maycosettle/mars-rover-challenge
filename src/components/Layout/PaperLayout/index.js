export { default as PaperLayout } from './PaperLayout';
