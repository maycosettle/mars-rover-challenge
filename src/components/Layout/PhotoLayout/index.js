export { default as PhotoLayout } from './PhotoLayout';
