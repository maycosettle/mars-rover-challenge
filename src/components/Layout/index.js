export * from './PageLayout';
export * from './PaperLayout';
export * from './PhotoLayout';
