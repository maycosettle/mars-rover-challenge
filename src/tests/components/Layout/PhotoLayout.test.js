import { shallow } from 'enzyme';
import { PhotoLayout } from 'components/Layout';

describe('tests on <PhotoLayout />', () => {
  const wrapper = shallow(<PhotoLayout />);

  test('Should render PhotoLayout correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
