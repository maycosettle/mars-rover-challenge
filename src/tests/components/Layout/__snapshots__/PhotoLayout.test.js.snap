// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on <PhotoLayout /> Should render PhotoLayout correctly 1`] = `
<div
  className="flex-1 flex flex-col justify-between px-8 divide-y divide-gray-200 sm:px-8"
>
  <div
    className="space-y-6 pt-2 pb-5"
  />
</div>
`;
